Sources/EmbraceCore/Session/SessionSpanUtils.swift

```diff
@@ -111,6 +117,17 @@ fileprivate extension SpanPayload {
             ))
         }
 
+        var mutableProperties = properties
+        mutableProperties.removeAll(where: { $0.key.contains("emb.user") })
+        attributeArray.append(
+            contentsOf: mutableProperties.map {
```

Comment:
Would recommend using `compactMap` here, then inside guard against the `emb.user` prefix and return `nil`.  Will remove an iteration through the loop.

Reply:
Actually, would best read using the `filter` method.

Reply:
done!